mailpoet/tests/unit/Subscribers/SubscriberSubscribeControllerUnitTest.php

```diff
@@ -74,6 +78,8 @@ public function testErrorGetsThrownWhenEmailFieldIsNotObfuscated() {
       $settings,
       $formsRepository,
       $statisticsFormsRepository,
+      $tagRepository,
```

Comment:
Maybe we can refactor this test class to use `\MailPoetTest::getServiceWithOverrides()` to avoid having to change all the places where `SubscriberSubscribeController` is instantiated whenever the signature of this class changes?

Reply:
Ah, sorry. You are right. Moving this PR to QA.